atlp-rwanda/commanders-rn-medica
# Feature: Sign in with google

#### What does this PR do?

This PR includes implementation of logging in with google OAuth.

#### Description of Task to be completed?

- Create OAuth keys
- Implement login with google
- Login user or create user using supabase. 

#### How should this be manually tested?

Download the built apk [here](https://expo.dev/accounts/8_commanders/projects/medica/builds/d19d911d-c571-4984-9c88-eec6f9d53fd3) and install it.

Navigate to the login page or signup page and click login with google. There will be a popup browser, enter your Gmail credentials and click continue. Wait for a second and you should be logged in.

#### Any background context you want to provide?

The OAuth keys are loaded from env variables. If you want to test i can send them to you privately or you can create yours for testing.

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
